src/java/org/apache/cassandra/io/sstable/StorageHandlerFactory.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.io.sstable;
+
+import org.apache.cassandra.db.Directories;
+import org.apache.cassandra.db.lifecycle.Tracker;
+import org.apache.cassandra.schema.TableMetadataRef;
+
+public interface StorageHandlerFactory
+{
+    StorageHandlerFactory INSTANCE = new StorageHandlerFactory() {};
```

Comment:
Ni: I'd called this `DEFAULT`; `INSTANCE` makes it sound like this the one instance of `StorageHandlerFactory`, but this may never get used at runtime if another factory is used.